linetools/spectra/xspectrum1d.py

```diff
@@ -111,24 +113,31 @@ def add_noise(self,seed=None,s2n=None):
         self.flux = self.flux.value + (rand * sig)*self.flux.unit
 
     #  Normalize
-    def normalize(self, conti, verbose=False, no_check=False):
+    def normalize(self, conti=None, verbose=False, no_check=False):
         """
         Normalize the spectrum with an input continuum
 
         Parameters
         ----------
         conti: numpy array
-          Continuum
+          Continuum. Use XSpectrum1D.co if None is given
         verbose: bool, optional (False)
         no_check: bool, optional (False)
           Check size of array?
         """
         # Sanity check
+        if conti is None:
+            if hasattr(self, 'co') and self.co is not None:
```

Comment:
Isn't self.co initialized to a bunch of NANs?
Maybe we should initialized to ones.